content/doc/book/managing/plugins.adoc

```diff
@@ -335,3 +335,46 @@ created/deleted to control the pinning behavior. If the `pinned` file is
 present, Jenkins will use whatever plugin version the user has specified.
 If the file is absent, Jenkins will restore the plugin to the default version
 on startup.
+
+== What is an implied dependency?
+
+Features are sometimes detached (or split off) from Jenkins core and moved into a plugin.
+
+Many plugins, like plugin:subversion[Subversion] and plugin:junit[JUnit], got their beginnings as Jenkins core functionalities.
+When a plugin was attached to the Jenkins core before being detached, it may or may not have used any of its functionality in other plugins that relied on that version of Jenkins.
+To guarantee that plugins do not fail when the functionality on which they rely is separated from Jenkins core, it is regarded to have a dependency on the detached plugin if it specifies a dependency on a version of Jenkins core prior to the split.
```

Comment:
```suggestion
When a plugin was attached to Jenkins core prior to being detached, it may or may not have used its full functionality with other plugins that relied on the same version of Jenkins.
To ensure that plugins do not fail when a functionality on which they rely is separated from Jenkins core, it is necessary to have a dependency in the detached plugin, if it specifies a dependency on a version of Jenkins core prior to the split.
```
update of syntax/content to try and make the text flow better, if this takes away from the message let me know and we can go from there!

Reply:
looks good